modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -55,8 +55,61 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Implement AsyncAPI Specifications
+
+Studio supports the implementation of AsyncAPI 2.6 specs. When creating an implementation project, Studio imports and scaffolds an AsyncAPI spec hosted on Exchange into an API interface that you can implement.
+
+For supported protocols, see xref:apikit::apikit-asyncapi-module-reference.adoc#supported-message-brokers[Supported Message Brokers] in APIkit for AsyncAPI Module Reference.
+
+=== Access AsyncAPI Implementation Features
+
+. Log in to Anypoint Platform.
+. Open Anypoint Code Builder landing page.
```

Comment:
Simone may ask for more info about where the landing page is. There's more info here that you could use. 

```suggestion
. Open Anypoint Code Builder landing page on your Anypoint Platform cloud host ([US](https://anypoint.mulesoft.com/codebuilder/)Leaving the Site or [EU](https://eu1.anypoint.mulesoft.com/codebuilder/)Leaving the Site).
```

Reply:
OK! Thanks!